content/blog/2020-03-16-teaching-online-on-short-notice/index.Rmarkdown

```diff
@@ -0,0 +1,215 @@
+---
+title: "Teaching online on short notice"
+description: |
+  Our experience teaching classes online may help people who have to do so on short notice.
+date: '2020-03-16'
+slug: teaching-online-on-short-notice
+categories:
+  - teach
+  - online
+tags: []
+author: [greg]
+photo:
+  url: https://www.fxguide.com/fxfeatured/dan-glass-deadpool-2-and-dneg/
+  author: fxguide
+---
+
+So here you are:
+you planned to teach your class or deliver your workshop in person,
+and now you have to do it online or not at all.
+Nobody is giving you time or money to make the change,
+and a hundred other things also need your attention.
+Where should you start, and what should you aim for?
+
+1.  *Don't change what you don't have to.*
+    Assignments, submission, grading, discussion forums:
+    if you're already doing them online,
+    don't change them now.
+    The same goes for content:
+    your slides and exercises may not be perfect,
+    but nothing ever is.
+
+1.  *Don't make assumptions about your learners' technology or circumstances.*
+    As [Elizabeth Wickes said][wickes-quote],
+    "Don’t presume people have {thing},
+    have great {thing},
+    have {thing} to themselves,
+    or even have a home where {thing} could be.
+    For each thing, fill in:internet, computer, monitor, attention, etc."
+
+1.  *Don't try to be (or to beat) recorded video.*
+    Even mediocre videos require time and production experience that you probably don't have right now,
+    so move your live teaching online instead of trying to record and edit yourself.
+    (If you *are* going to make videos,
+    please read Chen and Rabb's “[A Pattern Language for Screencasting][screencasting] first.)”
+
+1.  *Learn the song, then improvise over it.*
+    Good musicians don't just make things up on the fly:
+    they learn the song and then improvise around it.
+    *Great* musicians listen to their bandmates and their audience while they do this.
+    The best live teaching (whether in person or online) works the same way:
+    make a plan,
+    then vary your delivery depending on who your learners are,
+    what questions they ask,
+    and what you realize as you're speaking.
+
+1.  *Don't teach to empty chairs.*
+    We all rely on small social cues like nods and "mm hms" far more than we realize.
+    Teaching without these is really hard,
+    so choose a subset of your learners—four at least, half a dozen by preference—and
+    have them turn on video so that your hindbrain believes you actually have an audience.
+    This also promotes engagement:
+    after each break,
+    select a different subset of learners,
+    and always direct your questions to the people you can see.
+
+1.  *Start with a tour.*
+    Many features of your video conference tool will *not* be obvious to participants. 
+    Point out where they can toggle the chat,
+    change their participant view,
+    answer polls,
+    and so on.
```

Comment:
Add something like: "Prior to your live session, do a trial session where you join as a student (if you have a second device or a colleague/TA whose help you can enlist) and take some screenshots. Then, annotate these screenshots. Remember that the view of the web conferencing platform that you see as the host is likely different from what your students see."

Reply:
Good one - added.